tests/integration/jinad/test_simple_distributed/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.7.6-slim
+
+WORKDIR /
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y git \
+                                               curl
+
+COPY tests/distributed/requirements.txt /tests/distributed/requirements.txt
+
+RUN python -m pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir -r tests/distributed/requirements.txt
```

Comment:
this requirements may not be well addressed

Reply:
updated the `Dockerfile` to the right path `tests/integration/test_distributed_simple/`